### PR TITLE
Make the sidebar usable on mobile again

### PR DESCRIPTION
**Pull request by szaimen**
Fix https://github.com/nextcloud/calendar/issues/3289
| Before | After |
|---|---|
| image | image |

Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

**Comment by raimund-schluessler** [comment]
I think the class `calendar-app-sidebar` is no where applied, no?

**Comment by szaimen** [comment]
good point! Forgot to add the class. Fixed!

**Comment by raimund-schluessler** [comment]
I don't think that is a proper solution. It makes the whole calendar grid scroll, including the days header.

**Comment by szaimen** [comment]
Lets see what the others are thinking about this solution.

**Comment by szaimen** [comment]
This is now working as intended.
The trick was 
```scss
.app-sidebar-tab {
		overflow: unset !important;
		max-height: unset !important;
}
```

File: css/app-sidebar.scss [hunk 1]
```diff
@@ -83,22 +83,30 @@
...
 			padding-top: 14px;
-			position: absolute;
+			position: fixed;
 			bottom: 0px;
 			z-index: 2;
-			width: 97%;
-			left: 0px;
+			width: calc(27vw - 10px);
+			min-width: 300px - 10px;
+			max-width: 500px - 10px;
```
Comment (st3iny) [inline css/app-sidebar.scss:96]:
```suggestion
			width: calc(27vw - 11px);
			min-width: 300px - 11px;
			max-width: 500px - 11px;
```

The border is overlapped by 1 pixel.

sidebar-border-pixel

**Review by st3iny — DISMISSED** [review]
It works very well (tested with Chromium and Firefox). Thank you for fixing this :)

There is one minor thing I noticed. See below for specifics.

File: css/app-sidebar.scss (hunk 1 above)
Comment (szaimen) [reply css/app-sidebar.scss]: good find! Fixed :)

**Review by st3iny — APPROVED** [review]
LGTM

**Review by GretaD — APPROVED** [review]

**Comment by szaimen** [comment]
The failing tests are unrelated, afaics.

**Comment by szaimen** [comment]
> /compile amend /

I don't think that this is needed here

**Pull request merged**